Travis failing under Xcode 7.3

Can anyone fix that? I just upgraded the source code to support Xcode 7.3, Swift 2.2 and Swift 3 (as stated by Xcode) and now the Travis tests are failing. Thanks!

The Travis logs look like its failing to compile:

```swift
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:83:24: error: expected initial value after '='
        path: String = #file, _ function: String = #function, line: Int = #line) {
                       ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:83:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:83:82: error: expected parameter type following ':'
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:83:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:85:6: error: expected ')' in parameter
    }
     ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:82:30: note: to match this opening '('
    public class func verbose(@autoclosure message: () -> Any, _
                             ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:89:24: error: expected initial value after '='
        path: String = #file, _ function: String = #function, line: Int = #line) {
                       ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:89:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:89:82: error: expected parameter type following ':'
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:89:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:91:6: error: expected ')' in parameter
    }
     ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:88:28: note: to match this opening '('
    public class func debug(@autoclosure message: () -> Any, _
                           ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:95:24: error: expected initial value after '='
        path: String = #file, _ function: String = #function, line: Int = #line) {
                       ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:95:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:95:82: error: expected parameter type following ':'
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:95:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:97:6: error: expected ')' in parameter
    }
     ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:94:27: note: to match this opening '('
    public class func info(@autoclosure message: () -> Any, _
                          ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:101:24: error: expected initial value after '='
        path: String = #file, _ function: String = #function, line: Int = #line) {
                       ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:101:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:101:82: error: expected parameter type following ':'
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:101:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:103:6: error: expected ')' in parameter
    }
     ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:100:30: note: to match this opening '('
    public class func warning(@autoclosure message: () -> Any, _
                             ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:107:24: error: expected initial value after '='
        path: String = #file, _ function: String = #function, line: Int = #line) {
                       ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:107:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:107:82: error: expected parameter type following ':'
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:107:82: error: expected ',' separator
        path: String = #file, _ function: String = #function, line: Int = #line) {
                                                                                 ^
                                                                                ,
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:109:6: error: expected ')' in parameter
    }
     ^
/Users/travis/build/SwiftyBeaver/SwiftyBeaver/sources/SwiftyBeaver.swift:106:28: note: to match this opening '('
    public class func error(@autoclosure message: () -> Any, _
                           ^
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,5 @@
 language: objective-c
 xcode_project: SwiftyBeaver.xcodeproj # path to your xcodeproj folder
 xcode_scheme: SwiftyBeaver (iOS)
-osx_image: xcode7.2
-xcode_sdk: iphonesimulator9.2
+osx_image: xcode7.3
+xcode_sdk: iphonesimulator9.3

--- a/README.md
+++ b/README.md
@@ -7,7 +7,7 @@ Great for <b>development & release</b> due to its support for many (custom) logg
 ----
 
 ![Platform iOS 8+](https://img.shields.io/badge/Platform-iOS%208%2B-blue.svg) ![Platform Mac OS X 10.9+](https://img.shields.io/badge/Platform-Mac%20OS%20X%2010.9%2B-blue.svg) ![Platform](https://img.shields.io/badge/Platform-tvOS%209%2B-blue.svg) ![Platform](https://img.shields.io/badge/Platform-watchOS%202%2B-blue.svg)<br/>
-[![Language Swift 2](https://img.shields.io/badge/Language-Swift%202-orange.svg)](https://developer.apple.com/swift) [![Swift Package Manager compatible](https://img.shields.io/badge/Swift%20Package%20Manager-compatible-brightgreen.svg)](https://github.com/apple/swift-package-manager) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-brightgreen.svg)](https://github.com/Carthage/Carthage) [![Cocoapods compatible](https://img.shields.io/cocoapods/v/SwiftyBeaver.svg)]("https://cocoapods.org) [![Travis Build Status](http://img.shields.io/travis/SwiftyBeaver/SwiftyBeaver.svg)]("https://travis-ci.org/SwiftyBeaver/SwiftyBeaver)
+[![Language Swift 2](https://img.shields.io/badge/Language-Swift%202-orange.svg)](https://developer.apple.com/swift) [![Swift Package Manager compatible](https://img.shields.io/badge/Swift%20Package%20Manager-compatible-brightgreen.svg)](https://github.com/apple/swift-package-manager) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-brightgreen.svg)](https://github.com/Carthage/Carthage) [![Cocoapods compatible](https://img.shields.io/cocoapods/v/SwiftyBeaver.svg)](https://cocoapods.org) [![Travis Build Status](http://img.shields.io/travis/SwiftyBeaver/SwiftyBeaver.svg)](https://travis-ci.org/SwiftyBeaver/SwiftyBeaver)
 
 ----
 

